controllers/reviews/getAllReviews/getAllReviews.js

```diff
@@ -0,0 +1,12 @@
+import { Review } from '../../../models/review/index.js';
+
+export const getAllReviews = async (app) => {
+  app.get('/reviews', async (req, res, next) => {
```

Comment:
better to check is user is authenticated, we don't want to let anonymous users to trigger full DB search